### PR TITLE
Add ArbAssetLoader

### PR DESCRIPTION
**Pull request by SNNafi**
Add `ArbAssetLoader` for loading `.arb` files

### CODE REVIEW

**Comment by this-is-allan** [comment]
up

**Comment by SNNafi** [comment]
Any update on this @aissat ?

**Comment by bw-flagship** [comment]
@SNNafi THere are conflights in this branch

**Comment by SNNafi** [comment]
@bw-flagship , Fixed merge conflicts

File: README.md [hunk 1]
```diff
@@ -42,14 +43,13 @@ dependencies:
...
-  runApp(EasyLocalization(
-    child: MyApp(),
-    supportedLocales: [Locale('en', 'US'), Locale('ar', 'DZ')],
-    path: 'resources/langs/langs.csv',
-    assetLoader: CsvAssetLoader()
-  ));
+...void main(){
+runApp(EasyLocalization(
+child: MyApp(),
+supportedLocales: [Locale('en', 'US'), Locale('ar', 'DZ')],
```
Comment (bw-flagship) [inline README.md:49]: revert formating

**Review by bw-flagship — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SNNafi) [reply README.md:50]: Reverted

**Comment by bw-flagship** [comment]
@SNNafi I suppose something went wrong, you have far more changes than before

**Pull request closed without merging**